runtime/j9vm/java11vmi.c

```diff
@@ -1670,12 +1673,15 @@ JVM_SetBootLoaderUnnamedModule(JNIEnv *env, jobject module)
 #else /* JAVA_SPEC_VERSION >= 21 */
 				if (NULL == J9VMJAVALANGCLASSLOADER_UNNAMEDMODULE(currentThread, systemClassLoader->classLoaderObject)) {
 					J9Module *j9mod = NULL;
+#if defined(J9VM_OPT_SNAPSHOTS)
```

Comment:
We should only ifdef blocks of code such as this case, which impacts code block size / inlining / dead-code determination.

Reply:
Yeah, that makes sense.